Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "MIT"
 documentation = "https://docs.rs/scroll"
 description = "A suite of powerful, extensible, generic, endian-aware Read/Write traits for byte buffers"
 include = ["src/**/*", "Cargo.toml", "LICENSE", "README.md"]
+rust-version = "1.63"
```

Comment:
argh actually i didn't want to bump the msrv here because we don't have an actual requirement in scroll itself (only the dev deps)